.github/workflows/dcr-chromatic.yml

```
@@ -60,9 +60,13 @@ jobs:
         uses: actions/github-script@v7
         with:
           script: |
-            await github.rest.issues.removeLabel({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              issue_number: context.issue.number,
-              name: "run_chromatic"
-            })
+            try {
+              await github.rest.issues.removeLabel({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: context.issue.number,
+                name: "run_chromatic"
+              })
+            } catch (e) {
+              console.error(`An error ocurred: ${e}`);
+            }
```

Might be a bit tigher this way?

```suggestion
            await github.rest.issues.removeLabel({
              owner: context.repo.owner,
              repo: context.repo.repo,
              issue_number: context.issue.number,
              name: "run_chromatic"
            }).catch((error) => {
              // https://docs.github.com/en/rest/issues/labels?apiVersion=2022-11-28#remove-a-label-from-an-issue--status-codes
              if (error.status === 404) return console.warn("No such label!");
              throw error;
            })
```

---

Nice! I'd always felt unsure about `.catch`ing an `await`ed promise but the docs [^1] mention it too! Thanks for the tip!

[^1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function#rewriting_a_promise_chain_with_an_async_function

---

This catch also only accepts a known error with status 404. If Octokit or the network simply fails, we still want the step for fail, which this does 😃